vllm/entrypoints/llm.py

```diff
@@ -195,8 +199,50 @@ def __init__(
             mm_processor_kwargs=mm_processor_kwargs,
             **kwargs,
         )
-        self.llm_engine = LLMEngine.from_engine_args(
-            engine_args, usage_context=UsageContext.LLM_CLASS)
+
+        # TODO: should this be wrapped in a class?
+        if envs.VLLM_USE_V1:
+            engine_config = engine_args.create_engine_config()
```

Comment:
@njhill - im starting to think this stuff should be wrapped into `llmengine`

Reply:
@robertgshaw2-neuralmagic yeah I was thinking we might have a completely separate `LLM` class but that may be tricky if we want to be able to switch existing code with the env var.